packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -155,6 +157,14 @@ export class WorkflowsController {
 				throw new ApplicationError('No personal project found');
 			}
 
+			if (parentFolderId) {
+				let parentFolder = null;
+				try {
+					parentFolder = await this.folderService.getFolderInProject(parentFolderId, project.id);
+					await transactionManager.update(WorkflowEntity, { id: workflow.id }, { parentFolder });
+				} catch {}
```

Comment:
Is silently consuming the error here intentional?

Reply:
if they try to use wrong/invalid ID I just wanted to ignore as this comes from the URL, and not add a parent if this is the case.  I guess we can add a warning, but not sure how much value it brings

Reply:
Probably not much, leaving this as is is probably fine